dist/unshimmed-common-from-spark311.txt

```diff
@@ -8,10 +8,11 @@ com/nvidia/spark/RapidsUDF*
 com/nvidia/spark/Retryable*
 com/nvidia/spark/SQLPlugin*
 com/nvidia/spark/rapids/ColumnarRdd*
-com/nvidia/spark/rapids/GpuColumnVectorUtils*
 com/nvidia/spark/rapids/ExplainPlan.class
 com/nvidia/spark/rapids/ExplainPlan$.class
 com/nvidia/spark/rapids/ExplainPlanBase.class
+com/nvidia/spark/rapids/GpuColumnVectorUtils*
+com/nvidia/spark/rapids/GpuCoreDumpMsg*
```

Comment:
I think GpuCoreDumpMsg albeit tiny should be added to the sql-plugin-api module without adding to the tech debt of the classes maintained in this list. Even though GpuCoreDumpHandler as a whole cannot.